### PR TITLE
DOC-2005: fixing missing link

### PR DESCRIPTION
**Pull request by AMcFarlandCampbellR3**

### CODE REVIEW

File: content/en/docs/corda-enterprise/4.4/cordapp-advanced-concepts.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ This document provides the information you need in order to understand what happ
 Corda transactions evolve input states into output states. A state is a data structure containing: the actual data fact (that is expressed as a
 strongly typed serialized java object) and a reference to the logic (contract) that needs to verify a transition to and from this state.
 Corda does not embed the actual verification bytecode in transactions. The logic is expressed as a Java class name and a contract constraint
-(read more in: api-contract-constraints), and the actual code lives in a JAR file that is referenced by the transaction.
+(read more in [Contract Constraints](cordapps/api-contract-constraints)), and the actual code lives in a JAR file that is referenced by the transaction.
```
Comment (AMcFarlandCampbellR3) [inline content/en/docs/corda-enterprise/4.4/cordapp-advanced-concepts.md:46]:
```suggestion
(read more in [Contract Constraints](cordapps/api-contract-constraints)), and the actual code is contained in a .jar file that is referenced by the transaction.
```

File: content/en/docs/corda-enterprise/4.5/cordapp-advanced-concepts.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ This document provides the information you need in order to understand what happ
 Corda transactions evolve input states into output states. A state is a data structure containing: the actual data fact (that is expressed as a
 strongly typed serialized java object) and a reference to the logic (contract) that needs to verify a transition to and from this state.
 Corda does not embed the actual verification bytecode in transactions. The logic is expressed as a Java class name and a contract constraint
-(read more in: api-contract-constraints), and the actual code lives in a JAR file that is referenced by the transaction.
+(read more in [Contract Constraints](cordapps/api-contract-constraints)), and the actual code lives in a JAR file that is referenced by the transaction.
```
Comment (AMcFarlandCampbellR3) [inline content/en/docs/corda-enterprise/4.5/cordapp-advanced-concepts.md:43]:
```suggestion
(read more in [Contract Constraints](cordapps/api-contract-constraints)), and the actual code is contained in a .jar  file that is referenced by the transaction.
```

File: content/en/docs/corda-enterprise/4.6/cordapp-advanced-concepts.md [hunk 3]
```diff
@@ -40,7 +40,7 @@ This document provides the information you need in order to understand what happ
 Corda transactions evolve input states into output states. A state is a data structure containing: the actual data fact (that is expressed as a
 strongly typed serialized java object) and a reference to the logic (contract) that needs to verify a transition to and from this state.
 Corda does not embed the actual verification bytecode in transactions. The logic is expressed as a Java class name and a contract constraint
-(read more in: api-contract-constraints), and the actual code lives in a JAR file that is referenced by the transaction.
+(read more in [Contract Constraints](cordapps/api-contract-constraints)), and the actual code lives in a JAR file that is referenced by the transaction.
```
Comment (AMcFarlandCampbellR3) [inline content/en/docs/corda-enterprise/4.6/cordapp-advanced-concepts.md:43]:
```suggestion
(read more in [Contract Constraints](cordapps/api-contract-constraints)), and the actual code is contained in a .jar  file that is referenced by the transaction.
```

File: content/en/docs/corda-enterprise/4.7/cordapp-advanced-concepts.md [hunk 4]
```diff
@@ -40,7 +40,7 @@ This document provides the information you need in order to understand what happ
 Corda transactions evolve input states into output states. A state is a data structure containing: the actual data fact (that is expressed as a
 strongly typed serialized java object) and a reference to the logic (contract) that needs to verify a transition to and from this state.
 Corda does not embed the actual verification bytecode in transactions. The logic is expressed as a Java class name and a contract constraint
-(read more in: api-contract-constraints), and the actual code lives in a JAR file that is referenced by the transaction.
+(read more in [Contract Constraints](cordapps/api-contract-constraints)), and the actual code lives in a JAR file that is referenced by the transaction.
```
Comment (AMcFarlandCampbellR3) [inline content/en/docs/corda-enterprise/4.7/cordapp-advanced-concepts.md:43]:
```suggestion
(read more in [Contract Constraints](cordapps/api-contract-constraints)), and the actual code is contained in a .jar  file that is referenced by the transaction.
```

File: content/en/docs/corda-enterprise/4.8/cordapp-advanced-concepts.md [hunk 5]
```diff
@@ -40,7 +40,7 @@ This document provides the information you need in order to understand what happ
 Corda transactions evolve input states into output states. A state is a data structure containing: the actual data fact (that is expressed as a
 strongly typed serialized java object) and a reference to the logic (contract) that needs to verify a transition to and from this state.
 Corda does not embed the actual verification bytecode in transactions. The logic is expressed as a Java class name and a contract constraint
-(read more in: api-contract-constraints), and the actual code lives in a JAR file that is referenced by the transaction.
+(read more in [Contract Constraints](cordapps/api-contract-constraints)), and the actual code lives in a JAR file that is referenced by the transaction.
```
Comment (AMcFarlandCampbellR3) [inline content/en/docs/corda-enterprise/4.8/cordapp-advanced-concepts.md:43]:
```suggestion
(read more in [Contract Constraints](cordapps/api-contract-constraints)), and the actual code is contained in a .jar  file that is referenced by the transaction.
```

File: content/en/docs/corda-enterprise/4.4/cordapp-advanced-concepts.md [hunk 6]
```diff
@@ -43,7 +43,7 @@ This document provides the information you need in order to understand what happ
 Corda transactions evolve input states into output states. A state is a data structure containing: the actual data fact (that is expressed as a
 strongly typed serialized java object) and a reference to the logic (contract) that needs to verify a transition to and from this state.
 Corda does not embed the actual verification bytecode in transactions. The logic is expressed as a Java class name and a contract constraint
-(read more in: api-contract-constraints), and the actual code lives in a JAR file that is referenced by the transaction.
+(read more in [Contract Constraints](cordapps/api-contract-constraints)), and the actual code is contained in a .jar file that is referenced by the transaction.
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.4/cordapp-advanced-concepts.md:46]:
```suggestion
(read more in [Contract Constraints](cordapps/api-contract-constraints)), and the actual code is contained in a `.jar` file that is referenced by the transaction.
```

File: content/en/docs/corda-enterprise/4.5/cordapp-advanced-concepts.md [hunk 7]
```diff
@@ -40,7 +40,7 @@ This document provides the information you need in order to understand what happ
 Corda transactions evolve input states into output states. A state is a data structure containing: the actual data fact (that is expressed as a
 strongly typed serialized java object) and a reference to the logic (contract) that needs to verify a transition to and from this state.
 Corda does not embed the actual verification bytecode in transactions. The logic is expressed as a Java class name and a contract constraint
-(read more in: api-contract-constraints), and the actual code lives in a JAR file that is referenced by the transaction.
+(read more in [Contract Constraints](cordapps/api-contract-constraints)), and the actual code is contained in a .jar  file that is referenced by the transaction.
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.5/cordapp-advanced-concepts.md:43]:
```suggestion
(read more in [Contract Constraints](cordapps/api-contract-constraints)), and the actual code is contained in a `.jar`  file that is referenced by the transaction.
```

File: content/en/docs/corda-enterprise/4.6/cordapp-advanced-concepts.md [hunk 8]
```diff
@@ -40,7 +40,7 @@ This document provides the information you need in order to understand what happ
 Corda transactions evolve input states into output states. A state is a data structure containing: the actual data fact (that is expressed as a
 strongly typed serialized java object) and a reference to the logic (contract) that needs to verify a transition to and from this state.
 Corda does not embed the actual verification bytecode in transactions. The logic is expressed as a Java class name and a contract constraint
-(read more in: api-contract-constraints), and the actual code lives in a JAR file that is referenced by the transaction.
+(read more in [Contract Constraints](cordapps/api-contract-constraints)), and the actual code is contained in a .jar  file that is referenced by the transaction.
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.6/cordapp-advanced-concepts.md:43]:
```suggestion
(read more in [Contract Constraints](cordapps/api-contract-constraints)), and the actual code is contained in a `.jar`  file that is referenced by the transaction.
```

File: content/en/docs/corda-enterprise/4.7/cordapp-advanced-concepts.md [hunk 9]
```diff
@@ -40,7 +40,7 @@ This document provides the information you need in order to understand what happ
 Corda transactions evolve input states into output states. A state is a data structure containing: the actual data fact (that is expressed as a
 strongly typed serialized java object) and a reference to the logic (contract) that needs to verify a transition to and from this state.
 Corda does not embed the actual verification bytecode in transactions. The logic is expressed as a Java class name and a contract constraint
-(read more in: api-contract-constraints), and the actual code lives in a JAR file that is referenced by the transaction.
+(read more in [Contract Constraints](cordapps/api-contract-constraints)), and the actual code is contained in a .jar  file that is referenced by the transaction.
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.7/cordapp-advanced-concepts.md:43]:
```suggestion
(read more in [Contract Constraints](cordapps/api-contract-constraints)), and the actual code is contained in a `.jar`  file that is referenced by the transaction.
```

File: content/en/docs/corda-enterprise/4.8/cordapp-advanced-concepts.md [hunk 10]
```diff
@@ -40,7 +40,7 @@ This document provides the information you need in order to understand what happ
 Corda transactions evolve input states into output states. A state is a data structure containing: the actual data fact (that is expressed as a
 strongly typed serialized java object) and a reference to the logic (contract) that needs to verify a transition to and from this state.
 Corda does not embed the actual verification bytecode in transactions. The logic is expressed as a Java class name and a contract constraint
-(read more in: api-contract-constraints), and the actual code lives in a JAR file that is referenced by the transaction.
+(read more in [Contract Constraints](cordapps/api-contract-constraints)), and the actual code is contained in a .jar  file that is referenced by the transaction.
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.8/cordapp-advanced-concepts.md:43]:
```suggestion
(read more in [Contract Constraints](cordapps/api-contract-constraints)), and the actual code is contained in a `.jar`  file that is referenced by the transaction.
```

**Review by ivanterziev-r3 — CHANGES_REQUESTED** [review]
Just the missing backticks, the rest looks good

**Review by ivanterziev-r3 — APPROVED** [review]

**Pull request merged**